### ISSUE TITLE
Custom parse

### ISSUE THREAD

**Issue by Nerus87** [Challengermode/CoreRcon#14]
Hey,

I have a problem with creating my custom command parsing, there is no way to set flags or override the parsing method.

**Comment by xobust** [comment]
Hey! 
Sorry for the late reply. 
Can you give some more info? 

Have a look at this [example parser](https://github.com/Challengermode/CoreRcon/blob/master/src/CoreRCON/Parsers/Csgo/GameOverScore.cs)

**Comment by Nerus87** [comment]
Yeah, I've tried to recreate **status** ConVar in the context of checking Bots, SourceTV, do the better **status** ConVar for CSS/CSGO.
